openproblems/tasks/chromatin_potential/__init__.py

```diff
@@ -0,0 +1,8 @@
+from ... import utils
+from . import datasets, methods, metrics, checks
+
+_task_name = "Link gene expression with chromatin accessibility"
```

Comment:
Can this be made shorter? Three words or less is ideal, like "Chromatin accessibility prediction"

Reply:
Yes